lib/Froxlor/Cli/Action/ConfigServicesAction.php

```diff
@@ -83,6 +83,12 @@ private function createConfig()
 		$distros = glob($config_dir . '*.xml');
 		// tmp array
 		$distributions_select_data = array();
+
+		//read os-release
+		$os_dist = parse_ini_file('/etc/os-release', false);
+		$os_version = explode('.',$os_dist['VERSION_ID'])[0];
+		$os_default = "";
```

Comment:
default/fallback should stay "buster" (most commonly used) in case no default is found later on (as said, just fallback, should not happen)

Reply:
sure